scala.wake

```diff
@@ -8,6 +8,7 @@ def bloopInstall = "scala"
 def ivyCache = "ivycache"
 # Python wrapper script for invoking bloop
 def bloopWake = source "{here}/bloop_wake"
+def resources = "python/python/3.7.1", Nil # TODO add java
```

Comment:
We picked Python 3.5 because it's commonly included in older Linux distros default software repositories. Given that Wit is intended as a base tool that fetches other dependencies, I think maximal compatibility is important. I would rather keep this as Python 3.5 unless we find a compelling reason to require a newer version with a story for how to support older systems.

Reply:
And of course this is obviously not Wit so it doesn't matter as much, my bad.

I would rather just say this requires any Python 3.5 or greater, but if we have to pick a specific version, `3.7.1` is fine for this I think.

Reply:
All good on this then?

Reply:
Yeah